### PR TITLE
chore: Refactor ListingButton with deprecation of Listing V1

### PR DESCRIPTION
**Pull request by cbachmeier**
- Refactor and simplify ListingButton to remove no longer used listing state logic and states
- Remove old ListingModal from the Bag component
- Listing experience should be near identical from this deploy to prod

Screenshot:
refactoredListingButton

### CODE REVIEW

File: src/nft/components/profile/list/ListingButton.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  color: ${({ theme }) => theme.accentTextLightPrimary};
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  padding: 16px;
+  border-radius: 12px;
+  width: min-content;
+  border: none;
+  cursor: ${({ missingPrices }) => (missingPrices ? 'auto' : 'pointer')};
+  opacity: ${({ showResolveIssues, missingPrices }) => (showResolveIssues ? '1' : missingPrices && '0.3')};
```
Comment (JackShort) [inline src/nft/components/profile/list/ListingButton.tsx:27]: really nitty nit: should either be `(showResolveIssues ? '1' : missingPrices ? '0.3' : '1')` or `!showResolveIssues && missingPrices && '0.3')` not a huge fan of mixing different types of logical operator styles

File: src/nft/components/profile/list/ListingButton.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  // instantiate listings and collections to approve when user's modify input data
+  useEffect(() => {
+    const [newCollectionsToApprove, newListings] = getListings(sellAssets)
+    setListings(newListings)
+    setCollectionsRequiringApproval(newCollectionsToApprove)
+    setListingStatus(ListingStatus.DEFINED)
+  }, [sellAssets, setCollectionsRequiringApproval, setListingStatus, setListings])
+
+  // Find issues with item listing data
+  const [listingsMissingPrice, listingsBelowFloor] = useMemo(() => {
```
Comment (JackShort) [inline src/nft/components/profile/list/ListingButton.tsx:67]: some of these hooks might belong as their own hook -  might help readability of this file

**Review by JackShort — APPROVED** [review]
👋

File: src/nft/components/profile/list/ListingButton.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/profile/list/ListingButton.tsx:67]: I'm saving hook/listing logic refactoring for a subsequent PR

**Pull request merged**